src/components/UsingConfigCatClient.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="using-configcat-client">
+    <div class="flags-not-found" v-if="featureFlags === undefined || featureFlags.length === 0">
+      No feature flags found !
+    </div>
+    <div v-else class="feature-flag-cards-wrapper">
+      <div
+        class="feature-flag-card"
+        v-for="featureFlag in featureFlags"
+        :key="featureFlag.settingKey"
+      >
+        <div class="card-header">
+          <div class="feature-flag-name">{{ featureFlag.settingKey }}</div>
+        </div>
+        <div class="card-body">
+          <div class="feature-flag-value">{{ featureFlag.settingValue }}</div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { inject, onMounted, ref } from 'vue'
+
+// Import the ConfigCat client interface
+import { SettingKeyValue, type IConfigCatClient, type SettingValue } from 'configcat-vue'
+
+const configCatClient = inject<IConfigCatClient>('configCatClient')
+
+const featureFlags = ref<SettingKeyValue<SettingValue>[] | undefined>([])
+
+onMounted(async () => {
+  // Fetch feature flags
+  const response = await configCatClient?.getAllValuesAsync()
+  featureFlags.value = response
+})
```

Comment:
nit:
* You can omit the type argument of `SettingKeyValue` as the default type argument is `SettingValue.`
* I recommend throwing an error when `IConfigCatClient` is not available because that indicates a program error.

```suggestion
// Import the ConfigCat client interface
import type { SettingKeyValue, IConfigCatClient } from 'configcat-vue'

const configCatClient = inject<IConfigCatClient>('configCatClient')
  ?? (() => { throw new Error("ConfigCatPlugin was not installed."); })()

const featureFlags = ref<SettingKeyValue[] | undefined>([])

onMounted(async () => {
  // Fetch feature flags
  const response = await configCatClient.getAllValuesAsync()
  featureFlags.value = response
})
```